src/context/EarnManagerProvider/providers/yearn/api/api.ts

```diff
@@ -16,19 +16,52 @@ import {
   ApproveInput,
   APYInput,
   BalanceInput,
-  DepositInput,
-  WithdrawInput
+  TxEstimatedGasInput,
+  TxInput
 } from './yearn-types'
 
 export type ConstructorArgs = {
   adapter: ChainAdapter<ChainTypes.Ethereum>
   providerUrl: string
 }
 
+export type YearnVault = {
+  inception: 12022103
+  address: string
+  symbol: string
+  name: string
+  display_name: string
+  icon: string
+  token: {
+    name: string
+    symbol: string
+    address: string
+    decimals: number
+    display_name: string
+    icon: string
+  }
+  tvl: {
+    total_assets: number
+    price: number
+    tvl: number
+  }
+  apy: {
+    net_apy: number
+  }
+  endorsed: boolean
+  version: string
+  decimals: number
+  type: string
+  emergency_shutdown: boolean
+}
+
+const lower = (str: string) => str.toLowerCase()
```

Comment:
kinda nicer to use `lodash/toLower` rather than write our own utility in a provider

Reply:
Done.